t/lib/t/MusicBrainz/Server/Controller/User/Reverify.pm

```diff
@@ -31,12 +31,12 @@ test all => sub {
    my @emails = $test->get_emails;
    my $reverify_email = shift @emails;
    is($reverify_email->{headers}{To}, 'new_email@example.com', 'email sent to right place');
-   is($reverify_email->{headers}{Subject}, 'Please verify your email address', 'email subject is correct');
+   is($reverify_email->{headers}{Subject}, 'Verify your email', 'email subject is correct');
    my $reverify_email_body = $reverify_email->{body};
    like($reverify_email_body, qr{http://localhost/verify-email.*}, 'email contains verify-email link');
 
-   $reverify_email_body =~ qr{http://localhost(/verify-email.*)};
-   my $reverify_email_path = $1;
+   $reverify_email_body =~ qr{\[http://localhost(/verify-email.*?)\]}ms;
+   my $reverify_email_path = ($1 =~ s/\R//gr);
```

Comment:
This is not new from this PR, but... shouldn't this URL depend on the configured server address? (this test fails locally for me because `[error] Couldn't connect to the renderer. (URL: http://localhost/login)`). Same for `Edit` and `Register` above.